stubs/3.2/queue.pyi

```diff
@@ -6,5 +6,6 @@ from typing import Any
 
 class Queue():
     def get(self, block: bool = True, timeout: float = None) -> Any: pass
+    def put_nowait(self, item: Any) -> None: pass
```

Comment:
Regarding an earlier comment that I can't find now:

"""
In stubs/3.2/traceback.pyi:

> @@ -3,7 +3,8 @@
>  import typing
>
>  # TODO signatures
> +def format_exception_only(etype, value): pass
Why did you remove this one?
"""

I added that in, rather than removing it.

And regarding this comment:

"""
> @@ -6,5 +6,6 @@ from typing import Any
>
>  class Queue():
>      def get(self, block: bool = True, timeout: float = None) -> Any: pass
> +    def put_nowait(self, item: Any) -> None: pass
While this works, I think you could give the class a type parameter and make that the type of item and the return type of get.
"""

This was just a small change to add an extra method that I needed. If I add type parameters, that would be a separate commit/pull request.

And in general - for now - this is my first pull request. I'm making only minor changes - that I needed for a project - and seeing how this pull request goes, before I make more pull requests.